tests/e2e/requires_aws_api.go

```diff
@@ -39,6 +40,11 @@ import (
 const testTagName = "testTag"
```

Comment:
super minor np, but maybe rename this to `testTagNamePrefix` to reduce confusion?

/lgtm